### DISCUSSION TITLE
Making QTable Locale-Aware

### DISCUSSION THREAD

**Discussion by fwiwDev** [quasarframework/quasar#9126] [General - Components / Directives / etc] [answered]
Hi,

Is there a way to make QTable locale-aware so that the "Records per page" is in another language?

Thank you!

-Q

**Comment by hawkeye64** [comment, marked as answer]
@fwiwDev You have said if it's for v1 or v2, so I will assume v2. Otherwise, check the v1 docs, which should be similar. If you are changing the locale, you should be aware of the [Quasar Language Packs](https://next.quasar.dev/options/quasar-language-packs).
